unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2525,6 +2488,69 @@ abstract public class BaseFPSAgentController : MonoBehaviour {
             }
         }
 
+
+        /*
+        Naively parents one sim object under another game object. The child object
+        will become kinematic, this is necessary as parenting non-kinematic rigidbodies
+        can cause all sorts of issues.
+
+        Note that this function is "dangerous" in that it can have unintended interactions
+        with other actions. Please only use this action if you understand what you're doing.
+        */
+        public void ParentObject(string parentId, string childId) {
```

Comment:
Can this function result in a child having multiple parents that are disconnected? Basically, currently we will only have multiple parents for a child if there is a parent-child relationship between the two parents. Is this a correct observation?

Reply:
The object hierarchy in Unity is a tree so every object can only ever have a single parent. All this function really does is ensure that two objects will move together.